block_iagora.php

```diff
@@ -47,35 +47,43 @@ public function get_content() {
         }
 
         $this->content = new stdClass();
-        $this->content->footer = '';
-
         $iframeurl = isset($this->config->iframeurl) ? $this->config->iframeurl : '';
```

Comment:
Indeed, the `iframeurl` is also used in other files. However, its purpose has changed to referencing the Copilot Token Endpoint URL there, too. I think updating its name would make our code easier to read and more self-explanatory to avoid future confusion.

Reply:
quite, I'll modify it

Reply:
I added the moodle message icon directly to the template, avec `{{#pix}} i/edit, core {{/pix}}`